docs/INSTALL.md

```diff
@@ -209,6 +209,23 @@ under plain Windows.
 aka Windows Subsystem for Linux if that's what you prefer. Installation is just like any
 other Linux system.
 
+### Native Package managers
+
+Some people have created releases for d2 on windows package managers based on the prebuilt releases!
```

Comment:
Please edit copy to "The following are community-managed Windows releases of d2 on popular package managers"